cache/src/test/kotlin/com/dropbox/android/external/cache4/testutil/ConcurrencyTestRule.kt

```diff
@@ -9,26 +9,26 @@ import org.junit.runners.model.Statement
  */
 class ConcurrencyTestRule : TestRule {
     private var error: Throwable? = null
-    private var attemptIndex = 0
+    private var attempts = 0
 
     override fun apply(
         base: Statement,
         description: Description
     ): Statement {
         val annotation: ConcurrencyTest =
             description.getAnnotation(ConcurrencyTest::class.java) ?: return base
-        val times: Int = annotation.retries
-        require(times > 0) { "Number of retires must be greater than 1." }
+        val maxRetires: Int = annotation.retries
```

Comment:
@ychescale9 there is the same typo on the variable name here as well :) `retires`

Reply:
:man_facepalming: Thanks! And no I'm not considering retirement just yet...